src/auth/src/credentials/util/jws.rs

```diff
@@ -41,13 +48,7 @@ pub struct JwsClaims<'a> {
 
 impl JwsClaims<'_> {
     pub fn encode(&self) -> Result<String> {
```

Comment:
Did you consider `encode(&self, now: OffsetDateTime)` ?

I think that would lead to less "engineering" in the test.